### ISSUE TITLE
Is that possible to record all the unexpected error message using logger?

### ISSUE THREAD

**Issue by ArreCapClassique** [Delgan/loguru#738]
It works fine when i try to output specific message using logger.info and logger.success and so on. But there are still other errors i can't foresee in advance such as connection error, which are outputted in python default way, not loguru one. Is that possible to record all the error message displayed in the terminal using loguru and output them into a file using logger.add? I'm not using try...except... in my code.

**Comment by Delgan** [comment]
It depends on how these messages are sent.

If the messages are emitted through standard `logging` library, you'll need to [use an `InterceptHandler`](https://github.com/Delgan/loguru#entirely-compatible-with-standard-logging).
If the messages are directly written to `sys.stdout` or `sys.stderr`, you'll need to [redirect standard output](https://loguru.readthedocs.io/en/stable/resources/recipes.html#capturing-standard-stdout-stderr-and-warnings).

**Comment by ArreCapClassique** [comment]
so if i wish to output all error messages no matter they are emitted through standard logging library or directly written to sys.stdout or sys.stderr, i have to write a new function to combine InterceptHandler and standard output capture to make them all outputted by loguru?

**Comment by Delgan** [comment]
Yes, by combining the two solutions you will certainly capture everything that is emitted by third-party modules.

**Comment by ArreCapClassique** [comment]
in view of my coding ability, it would be extremely hard for me to combine them on my own. anyway, thx for your answering.

**Comment by ArreCapClassique** [comment]
i searched the internet and found several articles related to taking over the system log output. idk if the code they provided can be used directly or applied to most of the situation without much modification.
https://medium.com/1mgofficial/how-to-override-uvicorn-logger-in-fastapi-using-loguru-124133cdcd4e
https://segmentfault.com/a/1190000041784538/en

**Comment by Delgan** [comment]
You can find example implementations in the links I have provided.
